cs/ajax.texy

```diff
@@ -220,3 +220,35 @@ V takovém případě je nejjednodušší celý ten seznam obalit ještě jední
 \--
 
 Totéž platí i pro mazání. Sice by se dal nějak poslat prázdný snippet, jenže v praxi jsou většinou seznamy stránkované a řešit úsporněji smazání jednoho plus případné načtení jiného (který se předtím nevešel) by bylo příliš složité.
+
+
+Odesílání AJAXu na signál v komponentě
+======================================
+
+Máme-li komponentu s signálem, který přijímá parametry a chceme na ni vytvořit AJAXový request (například při změně hodnoty v inputu), tak nesmíme zapomenout na to že názvy parametrů se skládají z názvu komponenty a názvu parametru.
+
+Celý název parametru zjistíme pomocí metody v controlu `getParameterId`.
+
+/--html
+<script>
+var count = element.val();
+var id = element.attr("data-id");
+
+$.nette.ajax({
+	'url': {link changeCountBasket!},
+	'data': {
+		{$control->getParameterId('id')}: id,
+		{$control->getParameterId('count')}: count
+	}
+});
+</script>
+\--
+
+A handle metoda s odpovídajícími hodnotami v komponentě.
+
+/--php
+public function handleChangeCountBasket(...$values)
```

Comment:
Místo `...$values` bych tam dal `$id, $count`, nebo ne? Aby to souviselo s kodem výše.

Reply:
Ano, určitě, v původním jsem to měl, nevím proč jsem to měnil :)